core-aam/acacia/JSON-blockquote.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>role blockquote</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="role-map.json"></script>
+<body>
+  <div id=test role=blockquote>quote</div>
+
+  <script>
+    promise_test(async t => {
+
+      let result = await test_driver.test_accessibility_api_node('test',
+        rolemap['role-map-blockquote']
+      );
+      assert_equals(result, "match", "role=blockquote");
```

Comment:
This is a little awkward, `result` is either an error message containing all the statements that were incorrect, or, the string `"match"`